README.md

```diff
@@ -62,3 +62,62 @@ import { useTranslation } from 'language/LanguageProvider';
 const dict = useTranslation();
 dict.keyword; // or dict["keyword"]
 ```
+
+## Building
+
+### Preperation
+Building the application requires you to have [Node](https://nodejs.org/en/) and [Expo CLI](https://expo.io/tools#cli) installed.
+To get started simply clone the repo, open a terminal at project root and run `npm ci` to install all dependencies.
+You can now choose to build either a development build or an archive.
+
+
+### Development build
+
+To build an interactive development version of the application run `npm start` in the project root.
+This will build and launch and interactive version of the application that is accessable as either
+a website or an Android/iOS app through [Expo Go](https://expo.io/tools#client).
+
+### Building archive
+
+To build an Android or iOS archive run either `expo build:android` or `expo build:ios` in the project root.
+This will open a propt where you will be required to login to an expo account that will be responsible for the build.
+
+**Note:** If you're the publisher of the application remember that `expo build` will run `expo publish` automaticly if the build succeeds.
+
+When the build is done the archive will be available on your Expo account thourgh the link given by Expo CLI.
+For more build options please check out the Expo [documentation](https://docs.expo.io/distribution/building-standalone-apps/#3-start-the-build).
+
+
+## Translation
+
+Behaviour activation supports multiple luanguages and its easy to add additional languages.
+Adding a new language consists of two steps: 1. translations the actual app 2. tell the application that language exists.
+
+### Translating the application
+The first step in order to translate the application is to make a **copy** of the en english translation located at `src/language/en.tsx` and
+name it in accordance to the ISO 639-1 standard. (If you're unsure what language code to use you can find a list of them all over
+[here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)) You can now open the file and change the first occurance of `en` to your
+language code. You're now ready to start translating!
```

Comment:
```suggestion
language code. You are now ready to start translating!
```
More formally langugage, im not sure it´s nessesary tho

Reply:
Not really a super formal context and wanted to be consistent.